easybuild/easyconfigs/a/Albumentations/Albumentations-1.1.0-foss-2021b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonBundle'
+
+name = 'Albumentations'
+version = '1.1.0'
+
+homepage = 'https://albumentations.ai'
+description = "Albumentations is a Python library for fast and flexible image augmentations"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('CMake', '3.21.1'),
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('PyYAML', '5.4.1'),
+    ('scikit-image', '0.19.1'),
+    ('scikit-learn', '1.0.1'),
+    ('scikit-build', '0.11.1'),
```

Comment:
Also add `imgaug` as a dependency (it's optional for Albumentations, but we might as well include it)